src/xdocs/config_imports.xml

```diff
@@ -1645,6 +1645,31 @@ import java.util.stream.IntStream;
       </subsection>
 
       <subsection name="Examples" id="ImportOrder_Examples">
+        <p>
+          To configure the check:
+        </p>
+        <source>
+&lt;module name=&quot;ImportOrder&quot;/&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+//No violations Example
```

Comment:
Do violation and non-violation example separately

Reply:
ok